src/test/java/no/unit/nva/testutils/HandlerRequestBuilderTest.java

```diff
@@ -0,0 +1,95 @@
+package no.unit.nva.testutils;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import java.io.InputStream;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class HandlerRequestBuilderTest {
+
+    public static final String KEY = "key";
+    public static final String VALUE = "value";
+    public static final String BODY = "body";
+    public static final String PATH_PARAMETERS = "pathParameters";
+    public static final String QUERY_PARAMETERS = "queryParameters";
+    public static final String REQUEST_CONTEXT = "requestContext";
+
+    private ObjectMapper objectMapper;
+
+    @BeforeEach
+    public void setUp() {
+        objectMapper = new ObjectMapper();
```

Comment:
The reason for not using nva-commons JsonUtil for formatting is because we are emulating something else?

Reply:
We want to avoid circular dependency